bin/local-update.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+export CKAN_VERSION="2.11"
```

Comment:
Why hard-code this?

Reply:
this is for my local testing, its not used in cicd and i did not have ahoy/amzee installed

Reply:
It still could allow passing the version in, if `CKAN_VERSION` is already defined.

Reply:
ah, yep, it should be set if missing. I hard coded it.